runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/MetricsToPerStepNamespaceMetricsConverter.java

```diff
@@ -65,7 +67,10 @@ private static Optional<MetricValue> convertCounterToMetricValue(
       MetricName metricName,
       Long value,
       Map<MetricName, LabeledMetricNameUtils.ParsedMetricName> parsedPerWorkerMetricsCache) {
-    if (value == 0 || !metricName.getNamespace().equals(BigQuerySinkMetrics.METRICS_NAMESPACE)) {
+
```

Comment:
This is fine for now, but we probably want a better system for this check if we expand these metrics

Reply:
I agree.